noty-android/app/simpleapp/src/main/java/dev/shreyaspatil/noty/simpleapp/view/notes/adapter/NotesListAdapter.kt

```diff
@@ -48,6 +50,9 @@ class NotesListAdapter(
             with(binding) {
                 textTitle.text = note.title
                 textNote.text = note.note
+                pinnedIcon.run {
+                    if (note.isPinned) show() else hide()
+                }
```

Comment:
This code can be simplified in one-lines as:
```suggestion
                pinnedIcon.isVisible = note.isPinned
```

_`isVisible` is a kotlin extension property on `View`_